docs/connection-options.md

```diff
@@ -193,6 +195,8 @@ See [SSL options](https://github.com/mysqljs/mysql#ssl-options).
 
 * `applicationName` - A string visible in statistics and logs to help referencing an application to a connection (default: `undefined`)
 
+* `migrationsTransactionMode` - A string to control transactions for migrations, with cockroach this should be set to `each` (default: `all`)
```

Comment:
This only needs to be defined once.  I'll look into the default value for cockroach.

Reply:
this would need to be added in 3 places right?

- https://github.com/typeorm/typeorm/blob/master/src/commands/MigrationRunCommand.ts#L58
- https://github.com/typeorm/typeorm/blob/master/src/commands/MigrationRevertCommand.ts#L57
- https://github.com/typeorm/typeorm/blob/master/src/connection/Connection.ts#L200

Reply:
or even better in the connection options reader during normalisation:

https://github.com/typeorm/typeorm/blob/master/src/connection/ConnectionOptionsReader.ts#L177

```
            if (options.type === 'cockroachdb') {
                Object.assign(options, {
                    migrationsTransactionMode: 'each'
                })
            }
```

Reply:
@imnotjames shall I add this in the PR?

Reply:
No, let's just change the documentation here.